### DISCUSSION TITLE
Mass/Batch Translate

### DISCUSSION THREAD

**Discussion by FZ-Iskandar** [MohmmedAshraf/laravel-translations#83] [Ideas]
Being able to translate multiple keys at once would be greatly appreciated.

If that's a little bit too cumbersome to do, something easy to do might be a button to jump to the next untranslated string

![idea](https://github.com/MohmmedAshraf/laravel-translations/assets/8748656/4fc7e2e9-e5ae-4965-8355-316efdf7f6dd)

**Comment by MohmmedAshraf** [comment]
Hi @FZ-Iskandar , 

Actually when you save the translation it should redirect you to the next untranslated string

**Reply by FZ-Iskandar** [reply]
😅 Whops! didn't see that!!
